.gitignore

```diff
@@ -3,6 +3,9 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
+# package
+package-lock.json
```

Comment:
I believe this is incorrect. See https://stackoverflow.com/a/44210813/816663.

Our CI/CD should be using the exact versions of dependencies that were used at dev time.

src/front/package-lock.json is currently committed to the repo.

Reply:
See new issue #85